packages/gestalt/src/TypeaheadInputField.css

```diff
@@ -0,0 +1,23 @@
+
+.icon {
+  composes: marginEnd2 from "./boxWhitespace.css";
+  composes: noBorder from "./Borders.css";
+  composes: paddingX1 from "./boxWhitespace.css";
+  composes: paddingY1 from "./boxWhitespace.css";
+  composes: pointer from "./Cursor.css";
+  composes: right0 from "./Layout.css";
+  composes: transparentBg from "./Colors.css";
+  outline: none;
+  position: absolute;
+  top: 50%;
+  transform: translateY(-50%);
+}
+
+.inputRadius {
+  border-radius: 16px;
```

Comment:
@simkessy could we compose from `rounding4` in `packages/gestalt/src/Borders.css`?